js/main.js

```diff
@@ -1,9 +1,10 @@
 import { generateData } from './generate-data.js';
 import { getPopup } from './popup.js';
-import { switchFormsToDisabled } from './form.js';
+import { toggleFormToUnactive } from './form.js';
+import  './form.js';
```

Comment:
```suggestion
import  { initForm } './form.js';
```
ты тут тоже можешь все в одну функцию обернуть, чтобы у тебя точка входа была только в `main.js`